homeassistant/components/buienradar/camera.py

```diff
@@ -15,14 +15,20 @@
 
 CONF_DIMENSION = "dimension"
 CONF_DELTA = "delta"
+CONF_COUNTRY = "country_code"
 
-RADAR_MAP_URL_TEMPLATE = "https://api.buienradar.nl/image/1.0/" "RadarMapNL?w={w}&h={h}"
+RADAR_MAP_URL_TEMPLATE = (
+    "https://api.buienradar.nl/image/1.0/" "RadarMap{c}?w={w}&h={h}"
+)
 
 _LOG = logging.getLogger(__name__)
 
 # Maximum range according to docs
 DIM_RANGE = vol.All(vol.Coerce(int), vol.Range(min=120, max=700))
 
+# Multiple choice for available Radar Map URL
+MULTIPLE_CHOICE = vol.All(vol.Coerce(str), vol.In(["NL", "BE"]))
```

Comment:
I am not 100% sure this is consistent with the rest of hass, but `VALID_COUNTRY_CODE`?

You could even move this inline below like for CONF_DELTA.

Reply:
I taught too that it could be a good idea, but as Buineradar only handles Netherlands and Belgium, I taught it would be best to only allow multiple choice for those 2 countries.

Reply:
I think the name `MULTIPLE_CHOICE` is not descriptive :) – that is why I think an alternative would be clearer.

What about `SUPPORTED_COUNTRY_CODES`?

Reply:
Right good idea, just updated it, uploading all in a few minutes along with the test !